frontend/src/scenes/events/EventsVolumeTable.tsx

```diff
@@ -26,7 +26,8 @@ export function EventsVolumeTable(): JSX.Element {
                     </Tooltip>
                 )
             },
-            render: (item: EventUsageType) => humanizeNumber(item.volume),
+            // eslint-disable-next-line react/display-name
+            render: (item: EventUsageType) => <span className="ph-no-capture">{humanizeNumber(item.volume)}</span>,
```

Comment:
context: https://stackoverflow.com/questions/55620562/eslint-component-definition-is-missing-displayname-react-display-name